portal-api/src/constants/email-template-ids.js

```diff
@@ -3,11 +3,13 @@ const BLOCKED = '82506983-cb85-4f33-b962-922b850be7ac';
 const UNBLOCKED = '44959d08-6389-4f27-a6be-2faae8bea711';
 const PASSWORD_UPDATE = '41235821-7e52-4d63-a773-fa147362c5f0';
 const UPDATE_STATUS = '718beb52-474e-4f34-a8d7-ab0e48cdffce';
+const SIGN_IN_LINK = '2eab0ad2-eb92-43a4-b04c-483c28a4da18';
```

Comment:
I've just realised that
> If you did not create an account for UKEF export insurance, please contact us.
is definitely _not_ the right text - that's talking about EXIPS instead of DTFS. Hopefully when you get a response about the footer it will resolve this too - if not then we should get some new text for that
  

Reply:
This has been changed to `If you did not create an account for UKEF Digital Trade Finance, please contact us.`